chromium_src/content/public/browser/overlay_window.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_OVERLAY_WINDOW_H_
+#define BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_OVERLAY_WINDOW_H_
+
+#include "third_party/abseil-cpp/absl/types/optional.h"
```

Comment:
let's use `std::optional` in new code. we cleaned up existing `absl::optional` already